tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -98,11 +217,18 @@
         Field(item="26CIV", name='OTHER_NON_SANCTION', type='number', startIndex=112, endIndex=113,
               required=True, validators=[]),
         Field(item="27", name='WAIVER_EVAL_CONTROL_GRPS', type='number', startIndex=113, endIndex=114,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.or_validators(validators.matches(9), validators.isEmpty())
```

Comment:
confused on this one - spreadsheet says "any alphanumeric character is fine" but this appears to test for only `9` or ` `? am i reading that right?